src/data/Nav.js

```diff
@@ -0,0 +1,18 @@
+export const items = [
+  {
+    name: "About Us",
+    link: "aboutus",
+  },
+  {
+    name: "Events",
+    link: "events",
+  },
+  {
+    name: "Board",
+    link: "board",
+  },
+  {
+    name: "Resources",
+    link: "resources",
```

Comment:
this is right, but can you change the folder in the app folder that says contact to resources --- it was originally contact before ashley changed the design haha